tests/stepDefinitions/loginContext.js

```diff
@@ -1,4 +1,36 @@
 const { Given, When, Then } = require("@cucumber/cucumber");
 const { expect } = require("@playwright/test");
+const loginBtnLocator = "input[value='Login']";
```

Comment:
IMHO, it is better to separate the related snippets like this:
```suggestion
const { expect } = require("@playwright/test");

const loginBtnLocator = "input[value='Login']";
```